modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -1241,6 +1241,16 @@
 				# infinite loop between 0.0 and 0.99
 				f = wrapf(f + 0.1, 0.0, 1.0)
 				[/codeblock]
+				[codeblock]
+				# infinite rotation (in radians)
+				angle = wrapf(angle + 0.1, 0.0, TAU)
+				[/codeblock]
+				Note: If you want to have a simple wrap behavior from 0.0 to 'n' (where 'n' is a positive float number) then it's better for performance to use [method fmod] method like [code]fmod(number, n)[/code].
```

Comment:
Might be better if it were phrased like this:

"Note: If you **just** want to ~have a simple~ wrap **between** 0.0 **and** [code]n[/code] (where [code]n[/code] is a positive float **value**)[...]"

Same goes for the other instance.

Reply:
@akien-mga WDYT ? (I'm not an english speaker, so I dont know)

Reply:
The suggested edit seems good yes.

Reply:
Fixed